modules/logging/outputs.tf

```diff
@@ -2,8 +2,4 @@ resource "local_file" "ec2_private_key" {
   filename          = "ec2.pem"
   file_permission   = "0600"
   sensitive_content = tls_private_key.ec2.private_key_pem
-}
-
-output "ssh_private_key" {
-  value = tls_private_key.ec2.private_key_pem
 }
```

Comment:
What was the reason we removed this?

Reply:
I don't remember doing it, but I think it should stay gone - the stuff above it is outputting the same thing in a better way anyway right?